src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Tree/Dependencies/SDKVersionTelemetryServiceComponent.SDKVersionTelemetryServiceInstance.cs

```diff
@@ -42,6 +42,10 @@ protected override Task InitializeCoreAsync(CancellationToken cancellationToken)
                 // Do not block initialization on reporting the sdk version. It is possible to deadlock.
                 Task.Run(async () =>
                 {
+                    // Wait for the project to be loaded so that we don't prematurally load the active configuration
+                    await _unconfiguredProjectTasksService.ProjectLoadedInHost
```

Comment:
Note, this is our version of this API and will be cancelled if the project is unloaded unlike CPS's IProjectAsyncLoadDashboard.